RasimKhusaenov/my_own_blog
# Move creating article to separate page

### Summary

- Creating article moved to separate page
- Added "New" button on articles page
- Fixed article content displaying

### How it works

![image](https://user-images.githubusercontent.com/35406008/180062785-38956822-f7d0-41bb-b13d-4a2009967c6c.png)
![image](https://user-images.githubusercontent.com/35406008/180062844-f5c38242-9b9c-4ebe-a711-229d5059461e.png)

### Test plan

List of steps to manually test introduced functionality:

- Log in as admin
- Go to https://my-own-blog-pr-77.herokuapp.com/
- Click "New" button
- Make sure that you are redirected to article creating page

### Deploy notes

Notes regarding deployment the contained body of work.
These should note any db migrations, ENV variables, services, scripts, etc.

### Checklist:


### References
- Resolves #76 Issue
